public/components/create_report/create_report.tsx

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import {
+    EuiButtonEmpty,
+    EuiDatePicker,
+    EuiSelect,
+    EuiFlexGroup,
+    EuiFlexItem,
+    EuiFormRow,
+    EuiButton,
+    EuiPage,
```

Comment:
Noticed that you added `@ts-ignore` to the same component `EuiPage` in this PR, https://github.com/opendistro-for-elasticsearch/kibana-reports/pull/14. What's the reason for different style

Reply:
Got an error in my IDE that `EuiPage` didn't exist even though everything was functioning fine, so I added it to silence the warning